tools/go.mod

```diff
@@ -68,12 +68,12 @@ require (
 	github.com/spf13/viper v1.15.0 // indirect
 	github.com/subosito/gotenv v1.6.0 // indirect
 	github.com/toqueteos/webbrowser v1.2.0 // indirect
-	go.mongodb.org/mongo-driver v1.3.5 // indirect
+	go.mongodb.org/mongo-driver v1.5.1 // indirect
```

Comment:
Why do we need this dependency? I do not recall there is any usage of MongoDB in the main project... 🤔

Reply:
this addiction is pulling  github.com/go-swagger/go-swagger